freqtrade/arguments.py

```diff
@@ -171,6 +187,27 @@ def backtesting_options(parser: argparse.ArgumentParser) -> None:
             metavar='PATH',
         )
 
+    @staticmethod
+    def edge_options(parser: argparse.ArgumentParser) -> None:
+        """
+        Parses given arguments for Backtesting scripts.
+        """
+        parser.add_argument(
+            '-r', '--refresh-pairs-cached',
+            help='refresh the pairs files in tests/testdata with the latest data from the '
+                 'exchange. Use it if you want to run your edge with up-to-date data.',
+            action='store_true',
+            dest='refresh_pairs',
+        )
+        parser.add_argument(
+            '--stoplosses',
+            help='defines a range of stoploss against which edge will assess the strategy'
+                 'the format is "min,max,step" (without any space).'
```

Comment:
maybe add a space between "strategy" and "the" ... 
currently comes out as follows:
```
  --stoplosses STOPLOSS_RANGE
                        defines a range of stoploss against which edge will
                        assess the strategythe format is "min,max,step"
                        (without any space).example:
                        --stoplosses=-0.01,-0.1,-0.001

```